content/en/docs/tasks/debug/debug-application/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Debugging Applications"
+description: Debugging common containerized application issues.
+weight: 20
+---
```

Comment:
If you can, write an overview for the tasks in this section. A couple of sentences would do.